plugins/archlinux/README.md

```diff
@@ -17,8 +17,10 @@ plugins=(... archlinux)
 | pacin        | `sudo pacman -S`                       | Install packages from the repositories                           |
 | pacins       | `sudo pacman -U`                       | Install a package from a local file                              |
 | pacinsd      | `sudo pacman -S --asdeps`              | Install packages as dependencies of another package              |
+| paclean      | `sudo pacman -Sc`                      | Clean out old and unused caches and packages                     |
 | pacloc       | `pacman -Qi`                           | Display information about a package in the local database        |
 | paclocs      | `pacman -Qs`                           | Search for packages in the local database                        |
+| paclr        | `sudo pacman -Scc`                     | Remove all files from the cache                                  |
```

Comment:
`-Sc` (1 `c`) removes packages that are no longer installed
`-Scc` (2 `c`) removes all files from cache

[pacman man page](https://archlinux.org/pacman/pacman.8.html#_sync_options_apply_to_em_s_em_a_id_so_a):
> ## Sync Options (apply to -S)
> -c, --clean
> >	Remove packages that are no longer installed from the cache as well as currently unused sync databases to free up disk space. When pacman downloads packages, it saves them in a cache directory. In addition, databases are saved for every sync DB you download from and are not deleted even if they are removed from the configuration file pacman.conf(5). **Use one --clean switch to only remove packages that are no longer installed; use two to remove all files from the cache.** In both cases, you will have a yes or no option to remove packages and/or unused downloaded databases.